src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -142,6 +161,22 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
   const [isRadioPlaying, setIsRadioPlaying] = useState(false);
   const { volume, setVolume } = useRadio(isRadioPlaying, sound);
 
+  const handleMute = useCallback(
+    (volume: number) => (volume !== 0 ? 0 : 100),
+    []
+  );
+  const toggleMute = useCallback(() => setVolume(handleMute), [
+    handleMute,
+    setVolume,
+  ]);
+
+  const volumeControlClassname = classNames(
+    "NavBar__menu--link NavBar__menu--volume",
```

Comment:
Already made few BEM/SCSS suggestions https://github.com/sparkletown/sparkle/pull/2010#pullrequestreview-737596416 

@alexey-petrov use best judgement as to which you find simplest and takes you least of time to change.
We can go away with more complex changes due to time constraints

Reply:
fixed 0700a1d

Reply:
Now I'm noticing inconsistent use `NavBar__menu-link` and `NavBar__menu--link` as they should unify under the former. 

Anyways, writing this as a manner of course, and as long as page UI looks good, I'd leave the following for the next PR that will make naming more consistent:

- `NavBar` <- name of the block (as per CSS practices and sparkle project common use)
- `menu-link` <- name of the element (as per plain CSS practices)
- `NavBar__menu-link` <- element name consisting (as per BEM/project practices of `BlockName__element-name`)

Then adding a modifier `--muted` or `--volume-muted` (notice use of adjective instead of noun as a rule of thumb):

- `NavBar__menu-volume--volume-muted`